content/blog/2016/2016-04-07-jenkins-community-survey-results-blog.adoc

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "Jenkins Community Survey Results: CD, Pipelines, and Star Wars"
+tags:
+- jenkins
+- survey
+- continuous delivery
+- pipelines
+- docker
+- star wars
+author: bvdawson
+---
+
```

Comment:
Well, you're still the author, but the introduction would explain who your are and what you do. For past guest post introductions, see e.g.:
* https://jenkins.io/blog/2016/03/15/jenkins-certification/
* https://jenkins.io/blog/2016/01/27/jenkins-world-call-for-papers/
* https://jenkins.io/blog/2016/01/10/beautiful-jenkins-dashboard/
* https://jenkins.io/blog/2015/12/03/pipeline-as-code-with-multibranch-workflows-in-jenkins/